### ISSUE TITLE
Mismatch in performance

### ISSUE THREAD

**Issue by Red-Fairy** [pantheon5100/DeACL#1]
Hello! I found your work rather inspiring so I ran your code but failed to obtain the results mentioned in your paper. I got only 43.13% auto-attack accuracy under SLF settings (I ran the evaluation code in your repo) and 45.44% under AFF settings (since you didn't provide AFF evaluation code, I use the code written by myself). Moreover, I found that under ALF settings (as mentioned in [1]), the auto-attack accuracy is only 42.70%, which is even lower than that of SLF settings. Could you provide any explanations on these phenomenon, or is there any bugs in the code? I acknowledge that I didn't modify anything except the data path in your code, but I cannot obtain the results mentioned in paper.

[1] Lijie et.al. When does contrastive learning preserve adversarial robustness from pretraining to finetuning? NeruIPS, 2021

**Comment by pantheon5100** [comment]
Hi there, please check out our latest update.
